### PR TITLE
Update win_multiple_suspicious_cli.yml

### PR DESCRIPTION
**Pull request by mlp1515**
Add contains in CommandLine condition

### CODE REVIEW

**Comment by frack113** [comment]
Hello,
As this is a change in the detection part , can you add the `modified` field .
Thanks

**Comment by mlp1515** [comment]
I don't understand where I have to add the field 
Sorry

**Comment by frack113** [comment]
```
author: juju4
date: 2019/01/16
modified: 2021/06/14      
tags:
```
it is optional but useful for managing rules on a siem.
I use it in the filter https://github.com/SigmaHQ/sigma/blob/ae06ebcae08863b72960e826ea524c21dfa793cb/tools/sigma/filter.py#L157

**Comment by mlp1515** [comment]
Ok it's done 
Thanhks

**Review by StevenDias33 — APPROVED** [review]

**Review by StevenDias33 — APPROVED** [review]

**Review by StevenDias33 — APPROVED** [review]

File: rules/windows/builtin/win_user_added_to_local_administrators.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ detection:
     selection:
         EventID: 4732
     selection_group1:
-        GroupName: 'Administrators'
+        GroupName|startswith: 'Administr'
```
Comment (Neo23x0) [inline rules/windows/builtin/win_user_added_to_local_administrators.yml:21]: What do you think of this approach?

Comment (mlp1515) [reply rules/windows/builtin/win_user_added_to_local_administrators.yml:21]: Hi,
Good, much more efficient.

File: rules/windows/builtin/win_user_added_to_local_administrators.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ detection:
...
     selection_group2:
         GroupSid: 'S-1-5-32-544'
```
Comment (frack113) [inline rules/windows/builtin/win_user_added_to_local_administrators.yml:23]: It is `TargetSid: 'S-1-5-32-544'`

File: rules/windows/builtin/win_user_added_to_local_administrators.yml (hunk 1 above)
Comment (frack113) [reply rules/windows/builtin/win_user_added_to_local_administrators.yml:21]: it is `TargetUserName|startswith: 'Administr'`

**Review by frack113 — CHANGES_REQUESTED** [review]
Check with : https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4732

**Review by frack113 — APPROVED** [review]

**Review by StevenDias33 — APPROVED** [review]

**Pull request merged**